### PR TITLE
Fix 0.6.0 compilation errors in external tests

### PR DESCRIPTION
**Pull request by erak**
Fixes https://github.com/ethereum/solidity/issues/7460.

### CODE REVIEW

File: test/externalTests/common.sh [hunk 1]
```diff
@@ -201,6 +201,7 @@ function clean
 function run_install
 {
     local init_fn="$1"
+
```
Comment (erak) [inline test/externalTests/common.sh]: Dummy change.

**Comment by stackenbotten** [comment]
There was an error when running `chk_coding_style` for commit `85f50408c7c165e4a18cb365a775b7d8fadcf463`:
```
Error: Trailing whitespace found:
 test/externalTests/colony.sh:36: 

```
Please check that your changes are working as intended.

**Comment by stackenbotten** [comment]
There was an error when running `chk_coding_style` for commit `d216676c4991f48bc4b7a7c7829b3e4c77301f76`:
```
Error: Trailing whitespace found:
 test/externalTests/gnosis.sh:37: 
 test/externalTests/gnosis.sh:40: 

```
Please check that your changes are working as intended.

**Comment by erak** [comment]
Update:

- Gnosis and Colony work now.
- Zeppelin is failing because of https://github.com/ethereum/solidity/issues/5424#issuecomment-530787767. The reason is, that they have an inheritance structure which would need either a solution such as the one mentioned by @ekpyron in #5424 or allowing calldata parameters in public functions and a subtle change to their contracts.
So the the structure is basically this:
```
interface IRelayRecipient {
    function acceptRelayedCall() external;
}
abstract contract GSNRecipient is IRelayRecipient {
    function acceptRelayedCall() external { ... }
}
abstract contract GSNRecipientSignature is GSNRecipient {
    function acceptRelayedCall() external {...}
}
contract GSNRecipientSignatureMock is GSNRecipient, GSNRecipientSignature {
    function acceptRelayedCall() external override(IRelayRecipient, GSNRecipientSignature) {
        /* wants to call GSNRecipient.acceptRelayedCall(), but can't */
    }
}
```

**Comment by axic** [comment]
I'd be in favour merging this even without zeppelin worked. Still better than without.

**Comment by erak** [comment]
@axic Please feel free to merge :-) I think @ekpyron starting working on the solution mentioned here: https://github.com/ethereum/solidity/issues/5424#issuecomment-530787767. I had the impression that it might be beneficial to keep this open, since it's currently the only test that triggers this behaviour. But since it's a prominent issue already, we could also just merge and fix Zeppelin later.

**Comment by ekpyron** [comment]
Well, I started looking into https://github.com/ethereum/solidity/issues/5424#issuecomment-530787767, but I don't think I can finish that in time for a soonish 0.6 release. Let's discuss that part on Monday - but I agree that we can go ahead and merge this as is for now.

**Comment by ekpyron** [comment]
Actually the structure is:
```
abstract contract IRelayRecipient {
    function acceptRelayedCall() external;
}
abstract contract GSNRecipient is IRelayRecipient {
}
abstract contract GSNRecipientSignature is GSNRecipient {
    function acceptRelayedCall() external override {...}
}
contract GSNRecipientSignatureMock is GSNRecipient, GSNRecipientSignature {
    function acceptRelayedCall() external override(IRelayRecipient, GSNRecipientSignature) {
        /* wants to call GSNRecipient.acceptRelayedCall(), but can't */
    }
}
```

Which is interesting in two ways:
1. This should actually work and be exactly the same if ``GSNRecipientSignatureMock`` inherited only from ``GSNRecipientSignature``, but not needlessly again from ``GSNRecipient`` (which is a base of ``GSNRecipientSignature`` anyways...
2. We could consider this a bug in our override logic - we could redefine ``override`` to only require to explicitly mention every contract in which there is a base function *implementation*, but not if the base contract only contains it without implementation... If there is no implementation in any base class, we'd still need to require a plain ``override`` though.

**Comment by ekpyron** [comment]
I just rebased this to retrigger the tests - they should all go through now.

**Review by ekpyron — APPROVED** [review]

**Pull request merged**